File: etudiant.py
class etudiant:
    def __init__(self, nom, prenom, age, classe):
        self.nom = nom
        self.prenom = prenom
        self.age = age
        self.classe = classe
        
    def display_info(self):
        print(f"Nom: {self.nom}")
        print(f"prenom: {self.prenom}")
        print(f"Age: {self.age}")
        print(f"Classe: {self.classe}")
        
    def get_classe_level(self):
        if self.grade >= 5:
            print("master deuxieme année")
        elif self.grade >= 4:
            print("master premiere année")
        else:
            print("etudiant en cycle licence")

File: etudiant.java
public class etudiant {
    public String nom;
    public String prenom;
    public String date_naissance;
    private String Classe;

    public etudiant(String nom, String prenom, String date_naissance, String classe) {
        this.nom = nom;
        this.prenom = prenom;
        this.date_naissance = date_naissance;
        this.classe = classe;
}
    public String getClasse() {
        return this.classe;
    }

    public String getnom() {
        return this.nom;
    }

    public String getprenom() {
        return this.prenom;
    }

    public String getdate_naissance() {
        return this.date_naissance;
    }

    public void setnom(String nom) {
        this.nom = nom;
    }

    public void setprenom(String prenom) {
        this.prenom = prenom;
    }

    public void setdate_naissance(String date_naissance) {
        this.date_naissance = date_naissance;
    }

    public void setclasse(String Classe) {
        this.Classe = Classe;
    }
}
